Source/Tools/FEXLoader/FEXLoader.cpp

```diff
@@ -404,10 +403,14 @@ int main(int argc, char** argv, char** const envp) {
   FEXCore::Profiler::Init();
   FEXCore::Telemetry::Initialize();
 
-  RootFSRedirect(&Program.ProgramPath, LDPath());
-  InterpreterHandler(&Program.ProgramPath, LDPath(), &Args);
+  // Program.ProgramPath is highly likely to be prefixed with FEX's rootfs. Get rid of that.
+  if (Program.ProgramPath.starts_with(LDPath())) {
+    Program.ProgramPath = Program.ProgramPath.substr(LDPath().size());
```

Comment:
Avoid the extra memory allocation:
```suggestion
    Program.ProgramPath.erase(0, LDPath().size());
```